recipes-bsp/u-boot/u-boot-distro-boot/uEnv.txt.in

```diff
@@ -109,6 +112,108 @@ bootcmd_boot=if test ${bootscript_debug} != 1; then \
                  fi; \
              fi || true
 
-bootcmd_run=run board_fixups && run check_rollback_needed && run set_bootargs && run set_fdt_path && \
+prog_fuses=if test -n "${fuse_prog_list}" && test "${fuse_status}" = "pending" || test "${fuse_status}" = "need-reboot"; then \
+               if test "${fuse_status}" = "pending"; then \
+                   env set index 1; \
```

Comment:
Shouldn't `index` be in `VARS_TO_DEL_EXTRA`?

Reply:
It is, or are you talking about something else?

Reply:
Oh, somehow overlooked it. Closing this.